app/services/gpo_confirmation_uploader.rb

```diff
@@ -12,7 +12,13 @@ def run
     upload_export(export)
     LetterRequestsToGpoFtpLog.create(ftp_at: @now, letter_requests_count: confirmations.count)
     clear_confirmations(confirmations)
+    analytics.gpo_confirmation_upload(
+      success: true, exception: nil, gpo_confirmation_count: confirmations.count,
+    )
   rescue StandardError => error
+    analytics.gpo_confirmation_upload(
+      success: false, exception: error.to_s, gpo_confirmation_count: 0,
```

Comment:
Any chance the stringified error includes sensitive values?

Reply:
Yeah, there is a risk of that if an unencrypted confirmation record gets inspected. I don't see a way for that to happen but that doesn't mean it won't. I'm not sure what the best way to balance that risk is and this error is already reported to NewRelic.